pages/linux/deluser.md

```diff
@@ -0,0 +1,20 @@
+# deluser
+
+> Delete a user from the system.
+> Note: all commands must be executed as root.
```

Comment:
```suggestion
> Note: all commands must be executed as root.
> More information: <https://manpages.debian.org/deluser.8.en.html>.
```

Reply:
Sure, we can add this. Not all people have debian tho, and I thought that man pages are kinda too obvious

Reply:
Well, we do that if there's no other page.

Reply:
If there's something better than the Debian manpage go ahead and use it.  We use man7.org when possible, then I guess the Debian manpages.